addons/source-python/packages/source-python/translations/strings.py

```diff
@@ -217,12 +217,6 @@ def _replace_escaped_sequences(given_string):
         # Return the replaced string
         return given_string
 
-    def get_strings(self, key, **tokens):
```

Comment:
Yes, everyone should probably pass the complete dict of tokens, but with current implementation of `get_strings` those tokens will get stuck in `LangStrings` for absolutely no reason.
Regarding  to `__getitem__`, if somebody gets the empty `TranslationStrings` instance that is stored in `LangStrings` and does something filthy with its tokens, it's their problem. Creating a new `TranslationStrings` instance in `__getitem__` and then another one while tokenizing it? That's too much of instantiating, I say. Would be nice to hear opinions from other contributors, too. @Mahi, I believe this method originates from your PR?

I also don't see any reason for keeping `LangStrings.get_strings` now when we have `TranslationStrings.tokenized` method.